gcc/config/arc64/atomic.md

```diff
@@ -192,7 +192,7 @@
    (atomicop:SDI (match_dup 1)
 		(match_operand:SDI 2 "register_operand" "")) ;; operand
    (match_operand:SDI 3 "const_int_operand" "")]	;; model
-  ""
+  "TARGET_ATOMIC"
```

Comment:
The predicate needs to reflect the ATOMIC_2 option for the llockd/scondd variants